dev-python/scikit-build/files/scikit-build-0.10.0-skip-sandbox-violation-tests.patch

```diff
@@ -0,0 +1,137 @@
+-def test_install_command():
+-    pass
+-
+-
+-def test_test_command():
+-    with push_dir():
+-
+-        tmp_dir = _tmpdir('test_test_command')
+-        project = "issue-274-support-default-package-dir"
+-        prepare_project(project, tmp_dir)
+-        initialize_git_repo_and_commit(tmp_dir, verbose=True)
+-
+-        try:
+-            with execute_setup_py(tmp_dir, ["test"], disable_languages_test=True):
+-                pass
+-        except SystemExit as exc:
+-            assert exc.code == 0
+-
+-        assert tmp_dir.join("test_hello.completed.txt").exists()
+diff --git a/tests/test_issue274_support_one_package_without_package_dir.py b/tests/test_issue274_support_one_package_without_package_dir.py
+deleted file mode 100644
+index 6612087..0000000
+--- a/tests/test_issue274_support_one_package_without_package_dir.py
++++ /dev/null
+@@ -1,27 +0,0 @@
+-
+-from . import (
+-    _tmpdir, execute_setup_py, initialize_git_repo_and_commit,
+-    prepare_project, project_setup_py_test, push_dir
+-)
+-
+-
+-@project_setup_py_test("issue-274-support-one-package-without-package-dir", ["install"], disable_languages_test=True)
+-def test_install_command():
+-    pass
+-
+-
+-def test_test_command():
+-    with push_dir():
+-
+-        tmp_dir = _tmpdir('test_test_command')
+-        project = "issue-274-support-one-package-without-package-dir"
+-        prepare_project(project, tmp_dir)
+-        initialize_git_repo_and_commit(tmp_dir, verbose=True)
+-
+-        try:
+-            with execute_setup_py(tmp_dir, ["test"], disable_languages_test=True):
+-                pass
+-        except SystemExit as exc:
+-            assert exc.code == 0
+-
+-        assert tmp_dir.join("test_hello.completed.txt").exists()
+diff --git a/tests/test_issue284_build_ext_inplace.py b/tests/test_issue284_build_ext_inplace.py
+deleted file mode 100644
+index 78dd3dd..0000000
+--- a/tests/test_issue284_build_ext_inplace.py
++++ /dev/null
+@@ -1,15 +0,0 @@
+-
+-import os
+-import sysconfig
+-
+-from distutils import sysconfig as du_sysconfig
+-
+-from . import project_setup_py_test
+-
+-
+-@project_setup_py_test("issue-284-build-ext-inplace", ["build_ext", "--inplace"], disable_languages_test=True)
+-def test_build_ext_inplace_command():
+-    assert os.path.exists('hello/_hello_sk%s' % sysconfig.get_config_var('SO'))
+-
+-    # See issue scikit-build #383
+-    assert os.path.exists('hello/_hello_ext%s' % du_sysconfig.get_config_var('SO'))
+diff --git a/tests/test_issue334_configure_cmakelists_non_cp1252_encoding.py b/tests/test_issue334_configure_cmakelists_non_cp1252_encoding.py
+deleted file mode 100644
+index 2285bfb..0000000
+--- a/tests/test_issue334_configure_cmakelists_non_cp1252_encoding.py
++++ /dev/null
+@@ -1,7 +0,0 @@
+-
+-from . import project_setup_py_test
+-
+-
+-@project_setup_py_test("issue-334-configure-cmakelist-non-cp1252-encoding", ["install"], disable_languages_test=True)
+-def test_install_command():
+-    pass
```

Comment:
Could you please give some details on why this patch is need following the guidelines of https://dev.gentoo.org/~vapier/clean-patches ?